dotcom-rendering/src/components/LoopVideoPlayer.tsx

```diff
@@ -109,7 +113,9 @@ export const LoopVideoPlayer = forwardRef(
 		}: Props,
 		ref: React.ForwardedRef<HTMLVideoElement>,
 	) => {
+		// Assumes that the video is unique on the page.
```

Comment:
This comment is not needed since this PR: https://github.com/guardian/dotcom-rendering/commit/159236c6f7180f624d373fadeaa3ea05366cbfa3

The uniqueId is unique for any looping video on the page.